pom.xml

```diff
@@ -53,20 +53,20 @@
         <org.apache.xmlgraphics.version>2.8</org.apache.xmlgraphics.version>
         <org.apache.avalon.framework.version>4.3.1</org.apache.avalon.framework.version>
         <hamcrest.version>2.2-rc1</hamcrest.version>
-        <hibernate.version>5.6.10.Final</hibernate.version>
+        <hibernate.version>5.6.15.Final</hibernate.version>
         <jaxb.api.version>2.3.3</jaxb.api.version>
         <jaxb.glassfish-runtime.version>2.3.9</jaxb.glassfish-runtime.version>
         <jaxb2-basics-runtime.version>1.11.1</jaxb2-basics-runtime.version>
         <jaxen.version>1.2.0</jaxen.version>
         <jhove.version>1.20.1</jhove.version>
         <myfaces.version>2.3.10</myfaces.version>
-        <mysql.version>8.0.31</mysql.version>
-        <poi.version>5.2.2</poi.version>
+        <mysql.version>8.0.33</mysql.version>
+        <poi.version>5.2.5</poi.version>
         <primefaces.extensions.version>8.0.5</primefaces.extensions.version>
         <saxon.version>9.9.1-8</saxon.version>
         <log4j.version>2.19.0</log4j.version>
         <junit.version>5.9.2</junit.version>
-        <elasticsearch.version>7.17.10</elasticsearch.version>
+        <elasticsearch.version>7.17.13</elasticsearch.version>
```

Comment:
GitHub informed me on my fork, that there are more issues with ElasticSearch before `7.17.23`. Maybe we should use `7.17.23`? See https://mvnrepository.com/artifact/org.elasticsearch/elasticsearch for the `7.17` releases for the security issues.

Reply:
Yes, sure. Thanks for the hint.

Reply:
Done now.